files/management/commands/clear_permission_cache.py

```
@@ -0,0 +1,127 @@
+            )
+        except Exception as e:
+            self.stdout.write(
+                self.style.ERROR(f'Error clearing cache: {e}')
+            )
+
+    def clear_cache_by_pattern(self, pattern):
+        """Clear cache entries matching a specific pattern."""
+        try:
+            if hasattr(cache, 'delete_pattern'):
+                normalized = (
+                    pattern if pattern.startswith(f"{CACHE_KEY_PREFIX}:")
```

Ruff (0.12.2)

Undefined name `CACHE_KEY_PREFIX`

(F821)